README.md

```diff
@@ -1,19 +1,58 @@
-Check this [presentation](https://docs.google.com/presentation/d/1LnjIQT7tTrk7V7zK8xPE4LW-R5lJbAAPvDVEvPU6_xA/edit) for more information.  
+This tutorial shows some scenarios related to the Cilium and Spire integration.
```

Comment:
This integration modifies the following components: cilium-agent, cilium-envoy and spire-agent.